.github/workflows/retrobar.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       uses: dotnet/nbgv@v0.4.0
 
     - name: Publish app
-      run: dotnet publish -p:PublishProfile=$env:Profile
+      run: dotnet publish -p:PublishProfile=$env:Profile -f netcoreapp3.1
```

Comment:
Is this needed, given it is defined in the publish profiles?

Reply:
Yes, for now you have to explicitly specify the platform if there is more than one defined in the TargetFrameworks section of the project file, otherwise nothing will be published.

Check this: https://github.com/dotnet/sdk/issues/11474

Reply:
Ah, that's unfortunate.